pilot/pkg/networking/core/v1alpha3/cluster_test.go

```diff
@@ -1310,6 +1310,71 @@ func TestClusterDiscoveryTypeAndLbPolicyRoundRobin(t *testing.T) {
 	g.Expect(c.GetClusterDiscoveryType()).To(Equal(&cluster.Cluster_Type{Type: cluster.Cluster_ORIGINAL_DST}))
 }
 
+func TestSlowStartConfig(t *testing.T) {
+	g := NewWithT(t)
+	testcases := []struct {
+		name     string
+		clusters []*cluster.Cluster
+	}{
+		{"roundrobin", buildTestClusters(clusterTest{
```

Comment:
one additional test that roundrobin without slowstart config- might have been done in other tests, but for completeness